### DISCUSSION TITLE
Feature Request: Notes and Business/Personal Classification on Tax report

### DISCUSSION THREAD

**Discussion by bcyng** [teslamate-org/teslamate#1258] [Ideas]
The Grafana Dutch tax report is provides almost everything that is needed for Australian (and other jurisdictions?) statutory tax requirements.
But also need the following:
- a field to enter notes to describe the purpose of the trip.
- Business/Personal classification

Could we add these fields?

**Comment by ajeckmans** [comment]
New tax laws in the Netherlands (Rapportageverplichting Werkgebonden Personenmobiliteit - https://www.rvo.nl/onderwerpen/rapportage-wpm) will also require more accurate readouts on what is business related and what is personal per January 1st 2024 for any employee of a business size 100 and up.
